Sources/Datadog/DatadogCore/Storage/Files/Directory.swift

```diff
@@ -95,7 +95,7 @@ internal struct Directory {
         try retry(times: 3, delay: 0.001) {
             try files().forEach { file in
                 let destinationFileURL = destinationDirectory.url.appendingPathComponent(file.name)
-                try? retry(times: 3, delay: 0.000_1) {
+                try? retry(times: 3, delay: 0.0001) {
```

Comment:
Causing:
> Number Separator Violation: Underscores should be used as thousand separator in large decimal numbers. (number_separator)

in swiftlint `0.50.1`. I assume it now only requires `_` on the left side of the dot.